tools/sigma/backends/ala.py

```diff
@@ -266,7 +274,7 @@ def generateMapItemNode(self, node):
                     if type(mapitem) == str:
                         result.append(mapitem)
                     elif callable(mapitem):
-                        result.append(mapitem(self.cleanValue(val)))
+                        result.append(mapitem(self.cleanValue(str(val))))
```

Comment:
We should be able to remove this change once the new class method is defined.

```suggestion
                        result.append(mapitem(self.cleanValue(val)))
```